src/parseMessageDefinition.js

```diff
@@ -111,7 +112,7 @@ const buildType = (lines: string[]): RosMsgDefinition => {
         throw new Error("Malformed line: " + line);
       }
       let value: any = matches[2];
-      if (type !== "string") {
+      if (type !== "string" || isJson) {
```

Comment:
```suggestion
      if (type !== "string") {
```

not sure what this is doing here?

Reply:
Ah this is back when an `isJson` property was attached to definitions with a `string` type. Can be removed now.